mm/src/overlays/actors/ovl_En_Tru_Mt/z_en_tru_mt.c

```diff
@@ -337,7 +338,7 @@ void func_80B76924(EnTruMt* this) {
 void func_80B76980(EnTruMt* this, PlayState* play) {
     Player* player = GET_PLAYER(play);
 
-    if (gSaveContext.minigameHiddenScore >= 10) {
+    if (gSaveContext.minigameHiddenScore >= CVarGetInteger("gEnhancements.Minigames.BoatArcheryHealth", 10)) {
```

Comment:
We're trying to keep CVars out of source whenever possible. Let's make this use a VB instead:

```suggestion
    if (GameInteractor_Should(VB_FAIL_BOAT_ARCHERY, gSaveContext.minigameHiddenScore >= 10)) {
```

Then in Archery.cpp, do something like:
```c++
#define HEALTH_CVAR_NAME "gEnhancements.Minigames.BoatArcheryHealth"
#define HEALTH_CVAR CVarGetInteger(HEALTH_CVAR_NAME, 10)

COND_VB_SHOULD(VB_FAIL_BOAT_ARCHERY, HEALTH_CVAR != 10, {
    *should = gSaveContext.minigameHiddenScore >= HEALTH_CVAR;
});
```

Reply:
I hear you on this, and I'm making the change as we speak. I wondered if an exception would be allowed here just to not make the default comparison redundant.